yeat/bandage/Snakefile

```diff
@@ -0,0 +1,82 @@
+# -------------------------------------------------------------------------------------------------
+# Copyright (c) 2022, DHS. This file is part of YEAT: http://github.com/bioforensics/yeat
+#
+# This software was prepared for the Department of Homeland Security (DHS) by the Battelle National
+# Biodefense Institute, LLC (BNBI) as part of contract HSHQDC-15-C-00064 to manage and operate the
+# National Biodefense Analysis and Countermeasures Center (NBACC), a Federally Funded Research and
+# Development Center.
+# -------------------------------------------------------------------------------------------------
+
+from glob import glob
+import os
+from pathlib import Path
+import re
+
+
+rule all:
+    input:
+        expand("analysis/bandage/{assembler}/.done", assembler=config["assemblers"])
+
+
+rule spades:
+    output:
+        "analysis/bandage/spades/.done"
+    input:
+        "analysis/spades/assembly_graph_after_simplification.gfa",
+        "analysis/spades/assembly_graph_with_scaffolds.gfa",
+        "analysis/spades/assembly_graph.fastg",
+        "analysis/spades/strain_graph.gfa"
+    params:
+        bandage=config["bandage"]
+    run:
+        for gfa in input:
+            filename = Path(gfa).stem
+            shell("{params.bandage} image {gfa} analysis/bandage/spades/{filename}.jpg")
```

Comment:
As noted in #24, we can use Bandage CLI to 
1) save bandage images, 
2) load a bandage plot on QT5, 
3) created a reduced subgraph of the input graph, and 
4) get a particular query and display that query only

In this PR, I took all of the `gfa` and `fastg` files found in all assemblers and converted them into `jpeg` files in `analysis/bandage/[assembler]`.

When using Bandage's CLI, there are only two main options used to display the graphs. Depending on what we want or what we are looking for, we can customize the bandage images by changing some visual effects.

The other two options of reducing the graph and viewing a specific query are interesting tools as well.

Reply:
I haven't played around with points 3 and 4, but I will get on it to show you guys some output examples.